package.json

```diff
@@ -80,7 +80,7 @@
   },
   "packageManager": "yarn@4.1.1",
   "engines": {
-    "node": "^18.20 || ^20.17 || >=22"
+    "node": "^20 || ^22 || >=24"
```

Comment:
We ought to support all past versions of 20.x and 22.x, not just whichever happens to be the current version right now. This change also simplifies the process for bumping these ranges the next time around (no need to look up the current version).